protocol/pom.xml

```diff
@@ -57,11 +57,18 @@
       <version>${bcprov.version}</version>
     </dependency>
 
+    <dependency>
+      <!-- jsoup HTML parser library @ https://jsoup.org/ -->
+      <groupId>org.jsoup</groupId>
+      <artifactId>jsoup</artifactId>
+      <version>1.13.1</version>
+    </dependency>
+    
```

Comment:
Demo folder lib dependencies shouldn't be added in git.
Possible issue with the pom.xml for respective component. Either add the lib to .gitignore or perform mvn clean before adding the files to git.

Reply:
My mistake. Removing lib dependencies.

Reply:
done